src/index.js

```diff
@@ -163,6 +165,203 @@ class Radar {
       }
     );
   }
+
+  static searchPlaces(latitude, longitude, radius, chains, categories, groups, limit, callback) {
+    const publishableKey = Cookie.getCookie(Cookie.PUBLISHABLE_KEY);
+
+    if (!publishableKey) {
+      if (callback) callback(STATUS.ERROR_PUBLISHABLE_KEY);
+
+      return;
+    }
+
+    const finalLimit = Math.min(limit, 100);
+    let qs = `latitude=${latitude}&longitude=${longitude}&radius=${radius}&limit=${finalLimit}`;
```

Comment:
Rather than building the query string in each function, let's do this in `http.js`. 

Maybe change the `body` param to `data`, and the params can be passed in as an object (and converted to qs for the correct http methods)

Reply:
hmm, not sure I agree with this 100% -- there's custom logic around certain parameters for the `qs`. For example, capping the limit at 100, turning arrays into strings, etc. I don't want to bake these rules into the request data transformation layer if they won't apply for every request

Reply:
yeah, it's a bit odd / clunky at the moment that for all the `GET` endpoints, I have to send an empty body `{}`, and we're also conflating passing data with passing the raw endpoint URL ✔️